charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.14.9
 kubeVersion: ">=1.25.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 7.8.23
+version: 7.8.24
```

Comment:
Is it better to bump minor version since we change the default value ...? 🤔

Reply:
Thank you for your review. I added a commit for this. 🙋 
https://github.com/argoproj/argo-helm/pull/3242/commits/aca5a9968425ebe1c27b157e8d558f5d7baeaacb